manual/core/statements/prepared/README.md

```diff
@@ -55,35 +55,69 @@ client                            driver                Cassandra
   |<--------------------------------|                     |
 ```
 
+Beyond saving a bit of parsing overhead on the server, prepared statements have other advantages;
+the `PREPARED` response also contains useful metadata about the CQL query:
+
+* information about the result set that will be produced when the statement gets executed. The
+  driver caches this, so that the server doesn't need to include it with every response. This saves
+  a bit of bandwidth, and the resources it would take to decode it every time.
+* the CQL types of the bound variables. This allows bound statements' `set` methods to perform
+  better checks, and fail fast (without a server round-trip) if the types are wrong.
+* which bound variables are part of the partition key. This allows bound statements to automatically
+  compute their [routing key](../../load_balancing/#token-aware).
+* more optimizations might get added in the future. For example, [CASSANDRA-10813] suggests adding
+  an "[idempotent](../../idempotence)" flag to the response.
+
+If you have a unique query that is executed only once, a [simple statement](../simple/) will be more
+efficient. But note that this should be pretty rare: most client applications typically repeat the
+same queries over and over, and a parameterized version can be extracted and prepared.  
```

Comment:
maybe add "for example in the typical REST application your `BoundStatement`  can have a bound variable `user-id`" or something similar?

Reply:
There are examples of parameters and binding further down the page, so I don't think we need to go into too much detail here.